test/unit/argument_parser/format_parse_validators_test.cpp

```diff
@@ -163,6 +175,28 @@ TEST(validator_test, integral_range_validator_success)
     EXPECT_TRUE((testing::internal::GetCapturedStderr()).empty());
     EXPECT_EQ(option_vector[0], -10);
     EXPECT_EQ(option_vector[1], 1);
+
+    // get help page message
+    option_vector.clear();
+    const char * argv7[] = {"./argument_parser_test", "-h"};
+    argument_parser parser7("test_parser", 2, argv7);
+    parser7.add_positional_option(option_vector, "desc", integral_range_validator<std::vector<int>>(-20,20));
+
+
+    testing::internal::CaptureStdout();
+    EXPECT_THROW(parser7.parse(), parser_interruption);
+    std::string stdout = testing::internal::GetCapturedStdout();
+    std::string expected = "test_parser"
+                           "==========="
+                           "POSITIONAL ARGUMENTS"
+                           "    ARGUMENT-1 List of INT (32 bit)'s"
+                           "          desc Value must be in range [-20,20]."
+                           "VERSION"
+                           "    Last update: "
+                           "    test_parser version: "
+                           "    SeqAn version: 3.0.0";
```

Comment:
Must this test be updated with every new SeqAn release?

Reply:
It's set by the SEQAN_VERSION_MACRO which will be updated every release and thus no extra maintenance is needed.

Reply:
But here you do not use the macro, but a hard-coded `3.0.0` instead...

Reply:
ah true. good catch